### ISSUE TITLE
Getting Shulkerbox contents from player.json file

### ISSUE THREAD

**Issue by MattUKZ** [Multiverse/Multiverse-Inventories#476]
### Information

* **Server version:** This server is running Paper version git-Paper-71 (MC: 1.18.1) (Implementing API version 1.18.1-R0.1-SNAPSHOT) (Git: 91f3eda)
You are 3 version(s) behind
Download the new version at: https://papermc.io/downloads
Previous version: git-Paper-18 (MC: 1.18)

* **Full output of `/mv version -p`:** https://paste.gg/p/anonymous/0dd291268eeb4361a95e3c25e91d947c

* **Server log:** https://gist.github.com/MattUKZ/62fe22232cd5437d2fcf634a4ab5151f

### Help request

**Problem**
When updating to 1.18 I lost the backup I made of the 1.17 worlds/files etc and couldn't import all of the inventory data of players to the new multiverse config I did when recreating the basic setup I had for a survival, hardcore, and creative world for a few friends. I've managed to get back the inventory contents, off hand, and ender chest contents absolutely fine by importing the JSON file for the players from /plugins/worlds/world/player.json into Excel but I don't know how to decode the "internal" tag of the shulkerboxes in order to get the contents.

**What I have tried**
https://github.com/Multiverse/Multiverse-Inventories/issues/343
Without much luck I've attempted the comment from nicegamer7 on this issue, I got as far as decoding the internal tag in the Base64 to file but don't really understand where to go from there. If anyone has another solution or understands in simpler terms how to use this method that would be greatly appreciated.

**Screenshots**

**Comment by nicegamer7** [comment]
I can make a short video on how to do it if you'd like (sometime in the next week). Just give me an example I can use to make the video.

**Comment by MattUKZ** [comment]
That'd be absolutely great if you don't mind, I've linked one of the player.json files from one of the worlds - either the gray or light blue shulker box in the ender chest contents have items, really appreciate your help :) Let me know if you need any more files and i'll send them over. [Google Drive Folder](https://drive.google.com/drive/folders/1v7wdwFzimixsj2PNJ6ejNXBZ3hUIvSOE?usp=sharing)

**Comment by MattUKZ** [comment]
Do you reckon you'll get a chance to do this anytime soon @nicegamer7, don't wanna hurry you over the holidays just wondering whether to just try and remember the important items that were in them - thanks and no worries if not :)

**Comment by nicegamer7** [comment]
Yes, really sorry about the delay. I had a rough exam season, so I've just been taking some time off. But I'll get you that vid in a day or two!

**Comment by MattUKZ** [comment]
Awesome thank you so much man, wasn't meaning to hurry you understood you'd probably be busy with something - I really appreciate it though I hope the exams went well :)

**Comment by nicegamer7** [comment]
I went to make the video, but it looks like the player whose file you provided didn't have a shulker box 😅. No worries though, I did it with the example from #343. I see where you had difficulty, because I also had difficulty lol. It turns out (and I remember this now...), you have to remove backslashes from the string before converting to a file. Here's the video:

https://user-images.githubusercontent.com/8083772/148798426-5220be54-1b5d-4e23-9a13-fc931cd8fed3.mp4

**Comment by MattUKZ** [comment]
Thank you so much, and sorry I must have confused which player I intended to send - this feels like exactly what I tried to do but somehow has worked this time - I may have just made a silly mistake somewhere and seeing it on video helped. Thank you very much for the help and I hope the exams went well :)

**Comment by nicegamer7** [comment]
Awesome, glad to hear it helped! And thank you for your kind wishes :)

I'll be closing this now. Feel free to reopen or create a new issue if you need anything else.
